src/test/scala/com/asidatascience/configuration/DynamicConfigurationFromFileSpec.scala

```diff
@@ -66,6 +68,7 @@ with Inside {
     val parser = new TestConfigurationParser {}
     val configuration = newDynamicConfiguration(path, parser.parse)
     configuration.currentConfiguration shouldEqual None
+    ()
```

Comment:
Yes, the assertion returns an [`org.scalatest.compatible.Assertion`](https://github.com/scalatest/scalatest/blob/d40d278f2bb8e73431b942d87881fb6dbab75cb9/scalatest/src/main/scala/org/scalatest/compatible/Assertion.scala#L29).